pkg/cmd/kind/create/cluster/createcluster.go

```diff
@@ -27,6 +27,7 @@ import (
 
 	"sigs.k8s.io/kind/pkg/cluster"
 	"sigs.k8s.io/kind/pkg/cmd"
+	cmdutil "sigs.k8s.io/kind/pkg/cmd/util"
```

Comment:
pkg/cmd is already utils for cmd.
however, setting up this flag should also be internal, it's an implementation detail, NOT a public API

Reply:
make sense, i'll update it

Reply:
addressed

Reply:
intended `sigs.k8s.io/kind/pkg/cmd/internal/...` but this is OK, only cmd needs access to this